sources/packages/web/src/services/DesignationAgreementService.ts

```diff
@@ -32,9 +35,35 @@ export class DesignationAgreementService {
     );
   }
 
-  public async getDesignationsAgreements(): Promise<
-    GetDesignationAgreementsDto[]
-  > {
-    return ApiClient.DesignationAgreement.getDesignationsAgreements();
+  public async getDesignationsAgreements(
+    institutionId?: number,
+  ): Promise<GetDesignationAgreementsDto[]> {
+    return ApiClient.DesignationAgreement.getDesignationsAgreements(
+      institutionId,
+    );
+  }
+
+  public async getDesignationByStatus(
+    designationStatus: DesignationAgreementStatus,
+    searchCriteria?: string,
+  ): Promise<PendingDesignationDto[]> {
+    return ApiClient.DesignationAgreement.getDesignationByStatus(
+      designationStatus,
+      searchCriteria,
+    );
+  }
+
+  public async updateDesignationAgreement(
+    designationId: number,
+    designation: UpdateDesignationDto,
+  ): Promise<void> {
+    designation.locationsDesignations = designation.locationsDesignations?.filter(
+      location =>
+        !!(location.designationLocationId || location.approved === true),
```

Comment:
I believe that this would be more accurate as below
```typescript
(!!location.designationLocationId || location.approved === true)
```

Reply:
designationLocationId  is not used anymore so this is not required.